plugins/source/aws/policies_v1/views/log_metric_filter_and_alarm.sql

```diff
@@ -2,19 +2,27 @@ create or replace view view_aws_log_metric_filter_and_alarm as
 with af as (
   select distinct a.arn, a.actions_enabled, a.alarm_actions, m->'MetricStat'->'Metric'->>'MetricName' as metric_name -- TODO check
   from aws_cloudwatch_alarms a, jsonb_array_elements(a.metrics) as m
+),
+tes as (
+  select trail_arn from aws_cloudtrail_trail_event_selectors
+  where exists(
+    select * from jsonb_array_elements(event_selectors) as es
+    where es ->>'ReadWriteType' = 'All' and (es->>'IncludeManagementEvents')::boolean = TRUE
+  ) or exists(
+    select * from jsonb_array_elements(advanced_event_selectors) as aes
+    where not exists(select * from jsonb_array_elements(aes ->'FieldSelectors') as aes_fs where aes_fs ->>'Field' = 'readOnly')
+  )
 )
```

Comment:
These queries should be removed. `v23` will not have policies in the repo.

Reply:
So I can just revert the diff from this PR, right?

Reply:
8bae49f2cd5e58ec51b8717668c4e52d265a142c